web/src/pages/Login/Login.tsx

```diff
@@ -76,8 +86,7 @@ export default function Login() {
                         }
                     )
                     .then((e) => {
-                        setMessage('Connecting...', 'success');
-                        navigate('/accueil');
```

Comment:
why change this back ? Clara did a good job with the refactoring of the main page, please don't change it

Reply:
sorry, I didn't see I deleted some elements of Login(), it should be fix now